### ISSUE TITLE
no way to change language of vocabulary item

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#1238]
If a vocabulary item is assigned the wrong language, currently there's no way to reassign it the correct language other than to delete the item and add it again.

**Comment by ckjpn** [comment]
True, but deleting and adding an item again isn't too difficult and is quite intuitive, I think.

**Comment by alanfgh** [comment]
Delete-and-add-again is not intuitive for everybody. What's more, it does not preserve information (such as creation date). Furthermore, as more information is added to vocabulary items in the future (notes, etc.), deleting and adding them again will become more difficult.

**Comment by alanfgh** [comment]
This was mentioned on the Wall yesterday by [odexed](https://tatoeba.org/eng/wall/show_message/28356#message_28356).
